lms/envs/bok_choy.py

```diff
@@ -117,6 +117,11 @@
 FEATURES['SQUELCH_PII_IN_LOGS'] = False
 FEATURES['PREVENT_CONCURRENT_LOGINS'] = False
 FEATURES['ADVANCED_SECURITY'] = False
+
+FEATURES['ENABLE_MOBILE_REST_API'] = True  # Show video bumper in LMS
```

Comment:
This is needed to enable edxval_api. Bumper code checks for edxval_api presence in bumper_utils.is_bumper_enabled function